src/scss/_dropdown.scss

```diff
@@ -97,6 +96,5 @@
 	overflow-y: auto;
 	overflow-x: hidden;
 	max-height: $select-max-height-dropdown;
-	overflow-scrolling: touch;
```

Comment:
Unrecognised style, this might want putting back in and the style lint rule silencing.
It might get picked up by postCSS down the line and auto-prefixer might add --webkit

Reply:
When the build CSS is compiled this is not prefixed with `--webkit-`